src/sentry/web/urls.py

```diff
@@ -204,6 +205,11 @@
                     AuthOrganizationLoginView.as_view(),
                     name="sentry-auth-organization",
                 ),
+                re_path(
+                    r"^login/id/(?P<organization_id>[^/]+)/$",
```

Comment:
Since this is the first time `organization_id` is used as a parameter name in a route url; we will need to be mindful of the customer domain cases.

For example, when a user accesses `orgslug.sentry.io/auth/login/1234`, we need to validate that `orgslug` in the subdomain corresponds to the id `1234`.

Reply:
@dashed I assume this will be handled by your previous comment:
> If request.subdomain is not None, then it should have precedence over organization_id, and we should do a redirect to the correct organization_id.

Is there anything else to be done in addition to that?